src/components/Contacts/ContactDetails/ContactDontationsTab/PartnershipInfo/PartnershipInfo.tsx

```diff
@@ -0,0 +1,197 @@
+            )} ${contact?.pledgeFrequency ?? t('No Frequency Set')}`}
+          </LabelsAndText>
+          <LabelsAndText variant="subtitle1">{contact?.source}</LabelsAndText>
+        </Box>
+      </IconAndTextContainer>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          {contact?.pledgeReceived ? (
+            <CheckCircleOutline color="disabled" style={{ margin: 8 }} />
+          ) : (
+            <FiberManualRecordOutlined color="disabled" style={{ margin: 8 }} />
+          )}
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Commitment Recieved ')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.pledgeReceived ? t('Yes') : t('No')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <DateRangeOutlined color="disabled" style={{ margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Start Date')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.pledgeStartDate
+            ? DateTime.fromISO(contact?.pledgeStartDate).toLocaleString()
+            : t('No Date Available')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <DateRangeOutlined color="disabled" style={{ margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Last Gift')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.lastDonation?.donationDate
+            ? DateTime.fromISO(
+                contact?.lastDonation.donationDate,
+              ).toLocaleString()
+            : t('No Date Available')}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Lifetime Gifts')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {currencyFormat(
+            contact?.totalDonations ?? 0,
+            contact?.pledgeCurrency ?? 'USD',
+          )}
+        </LabelsAndText>
+      </IconAndTextContainerCenter>
+      <IconAndTextContainerCenter>
+        <IconContainer>
+          <Clear style={{ color: 'transparent', margin: 8 }} />
+        </IconContainer>
+        <LabelsAndText variant="subtitle1" color="textSecondary">
+          {t('Referred by ')}
+        </LabelsAndText>
+        <LabelsAndText variant="subtitle1">
+          {contact?.contactReferralsToMe.nodes
+            .map((referral) => {
+              return referral.referredBy.name;
+            })
```

Comment:
```suggestion
            .map((referral) => referral.referredBy.name)
```
Prettier may format this differently than I have it...